dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -1192,6 +1199,21 @@ internal class RumViewScopeTest {
         verifyZeroInteractions(mockWriter)
         assertThat(result).isSameAs(testedScope)
         assertThat(testedScope.activeActionScope).isSameAs(mockChildScope)
+
+        verify(mockDevLogHandler).handleLog(
+            eq(Log.WARN),
+            argThat {
+                contains("was dropped") &&
```

Comment:
I didn't go for the exact match on purpose

Reply:
why not asserting the whole string here ?

Reply:
because then I have to copy-paste it here (it is parametrized) and keep both in sync, but not a big deal if needed. Just imo we care about the context of the message only.

Reply:
if you use an internal constant for that message format in the other class then you can share it with the test. This is what we usually do in this case.

Reply:
yes, but the thing is that it is not constant, it is parametrized. I saw that in some places there is dedicated `format` method to format message with parameters. do you propose to extract it to the `format` method?

Reply:
Exactly ;) this is what I usually do.